stdlib/Random/test/runtests.jl

```diff
@@ -627,7 +627,7 @@ guardseed() do
     m = MersenneTwister(0)
     @test Random.seed!() === g
     @test Random.seed!(rand(UInt)) === g
-    @test Random.seed!(rand(UInt32, rand(1:10))) === g
+    @test Random.seed!(rand(UInt32, rand(1:8))) === g
```

Comment:
Out of curiosity, why this change?

Reply:
Because of the error for using too-long seeds. If we remove the error then this is ok.

Reply:
Ah right of course, for some reason I thought `1:10` was the sampler from which values were sampled.